common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -645,7 +647,7 @@ uint8_t wireless_utils::get_operating_class_max_tx_power(
 /**
  * @brief get operating class number by channel and channel bandwidth 
  *
- * @param channel current channel
+ * @param channel current channel (center channel for bandwidth 80 and 160)
```

Comment:
It's very easy to make a mistake if just the comment there explains that the caller must pass the center channel.
It might be better to rename the function to "get_operating_class_by_center_channel"

Reply:
But it doesn't always accept center channel. If the BW is 20/40 - it uses the starting channel. If the BW is 80/160 - it uses the center channel. How would you do it?

Reply:
OK - see https://github.com/prplfoundation/prplMesh/pull/938/commits/cf6c38031fb9c9dca108927f0a199d033d7e9a28 if its better